content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md

```diff
@@ -20,3 +20,16 @@ They distinguish between problems with the different types of queues and
 replace the `application_queue_full` alarm type, which has been removed in this release.
 See [Performance alarms](https://cumulocity.com/guides/apama/troubleshooting/#performance_alarms)
 in the *Streaming Analytics* guide for detailed information on the new alarms.
+
+
+### Improvements for EPL apps
+
+<!-- TODO: Add more info on new multi-tenant variant of Apama-ctrl microservice later -->
+
+A new EPL sample named "Create an alarm if a measurement exceeds a threshold value (multi-tenant)"
+can now be accessed from the EPL editor of the Streaming Analytics application.
```

Comment:
we don't call anything as EPL editor ? , perhaps EPL Apps editor ? 

also, the sample is not really in the editor but editor screen

perhaps we should just say, a new sample named "foo" is added to EPL Apps in streaming analytics application ?

Reply:
Hm, we're calling this thing "EPL editor" everywhere. For example at https://cumulocity.com/releasenotes/release-10-9-0/streaming-analytics-10-9-0/#10_9_0 or https://cumulocity.com/guides/10.15.0/apama/analytics-introduction/#apama-epl-apps 

I don't think that your suggestion makes the position of the samples any clearer. One might then think that they are shown as a card in the so-called "EPL app manager".

I suggest to either keep the existing text or make it even longer to say:
"... can now be accessed via the Samples button which is shown to the right of the EPL editor in the Streaming Analytics application."

What do you say?

Reply:
:D, lets just keep the existing one.

I think I was not thinking straight as its closer to my end of the day.

Reply:
:-)